libs/optimizer/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "optimizer",
+  "version": "0.0.1",
+  "dependencies": {
+    "igc-xc-score": "^1.7.0",
```

Comment:
moved in top level. But I thought that the dependencies should be located here if we want to publish the library.

Reply:
> But I thought that the dependencies should be located here if we want to publish the library.

I'm not entirely sure about this one but I hope that the package.json generated in the dist folder automatically figures out the deps. Otherwise it would be super easy to miss some when they are installed at the top level.

Reply:
Looks like I am wrong and deps actually need to be listed here.